.github/workflows/update-transitive-dependenies.yaml

```diff
@@ -35,8 +35,7 @@ jobs:
           labels: dependencies
     - name: Enable Pull Request Automerge
       if: steps.cpr.outputs.pull-request-operation == 'created'
-      uses: peter-evans/enable-pull-request-automerge@v3
-      with:
-        token: ${{ secrets.ZORGBORT_TOKEN }}
-        pull-request-number: ${{ steps.cpr.outputs.pull-request-number }}
-        merge-method: merge
+      run: gh pr merge --merge --auto "$pr_number"
+      env:
+        GH_TOKEN: ${{ secrets.ZORGBORT_TOKEN }}
+        pr_number: ${{ steps.cpr.outputs.pull-request-number }}
```

Comment:
I'm not certain about this plus the $pr_number in the `run` part. I don't think we need both and looking at the github docs we might not need either since this action targets the branch of the workflow.